.travis.yml

```diff
@@ -43,9 +43,10 @@ script:
     echo "Running ACNV pipeline";
     bash scripts/acnv_cromwell_tests/run_acnv_pipelines.sh;
   elif [[ $TEST_DOCKER == true ]]; then
-    echo "Building docker image with unit tests...";
-    cd scripts/docker/;
-    sudo bash build_docker.sh  -e `git rev-parse ${TRAVIS_BRANCH}` -s;
+    echo "Building docker image with unit tests..." ;
+    echo "${TRAVIS_BRANCH}" ;
+    sudo cp -rfp scripts/docker/Dockerfile . ;
+    sudo docker build -t broadinstitute/gatk-protected:${TRAVIS_BRANCH} --build-arg GITHUB_TAG=${TRAVIS_BRANCH} --build-arg GITHUB_DIR=\  . ;
```

Comment:
Dockerfiles usually go in the root of the repo. In my Jenkins I just move this to root and run a regular docker build. No script needed. And that's how it should run. Can we just get rid of the docker build script you have and run that way?